circe-v1/src/main/scala/dsl/circe/HasEncoder.scala

```diff
@@ -0,0 +1,20 @@
+package com.rallyhealth.vapors.v1
+
+package dsl.circe
+
+import io.circe.Encoder
+
+// TODO: Figure out how to mix and match DSLs
```

Comment:
This is effectively done in the [next PR](https://github.com/jeffmay/vapors/pull/68)

Reply:
I determined that this is not an issue. I am able to use type bounds to constrain subclasses, and then in the specific DSL class, I just pick a single type for which I can combine everything. DSLs are not created on the fly very often, if at all, so this isn't really an issue. I removed the TODO.